syntax = "proto3";

package firedetection;

// The greeting service definition.
service FireDetectionGRPC {
  // Sends a greeting
  rpc TakeAlarm(GetRequest) returns (ReturnResponse);
}

// The request message containing the user's name.

message GetRequest {
	string cameraID = 1;
	string percentFire = 2;
	string Time = 3;
}

message ReturnResponse {
	string message = 1;
}
